src/components/Rocket.jsx

```diff
@@ -4,50 +4,58 @@ import { useSelector, useDispatch } from 'react-redux';
 import { reserveRocket, cancelRocket } from '../redux/rocketsReducer';
 import './css/Rocket.css';
 
-Rocket.propTypes = {
-    rocket: PropTypes.shape({
-        id: PropTypes.number.isRequired,
-        rocket_name: PropTypes.string.isRequired,
-        description: PropTypes.string.isRequired,
-        flickr_images: PropTypes.arrayOf(PropTypes.string).isRequired,
-    }).isRequired,
-};
-
 function Rocket({ rocket }) {
   const rockets = useSelector((state) => state.rockets.rockets);
-  const { id, rocket_name, description, flickr_images,  } = rocket;
-  const [image, setImage] = useState(flickr_images[0]);
-    const dispatch = useDispatch();
+  const {
+    // eslint-disable-next-line camelcase
+    id, rocket_name, description, flickr_images,
+  } = rocket;
+  // eslint-disable-next-line camelcase
+  const [image] = useState(flickr_images[0]);
+  const dispatch = useDispatch();
+
+  const reserve = () => {
+    dispatch(reserveRocket(id));
+  };
 
-    const reserve = () => {
-        dispatch(reserveRocket(id));
-        setReserved(true);
-    };
- 
-    const cancel = () => {
-        dispatch(cancelRocket(id));
-        setReserved(false);
-    };
+  const cancel = () => {
+    dispatch(cancelRocket(id));
+  };
 
-    //check if rocket is reserved
-    const reserved = rockets.find((rocket) => rocket.id === id).reserved;
-        
+  // check if rocket is reserved
+  const { reserved } = rockets.find((rocket) => rocket.id === id);
 
   return (
     <div className="rocket-card">
+      {/* eslint-disable-next-line camelcase */}
       <img src={image} alt={rocket_name} />
       <div className="rocket-info">
-        <h2>{rocket_name}</h2> 
-        <p>{reserved? (<p className='reserve'>Reserved</p>):null}{description}</p> 
+        {/* eslint-disable-next-line camelcase */}
+        <h2>{rocket_name}</h2>
+        <p>
+          {reserved ? (<p className="reserve">Reserved</p>) : null}
+          {description}
+        </p>
```

Comment:
Hey, @niicode attached is a screenshot of what happens when the `Reserve rocket` button is clicked. I think to avoid this it would be better if you changed the parent <p> to a <div>.